README.md

```diff
@@ -738,6 +719,7 @@ will honor that instead, and will not do any automatic configuration.
 
 Compiling crypto on Red Hat systems
```

Comment:
I _think_ this is bit rotted - Erlang handles this situation "out of the box" now

Reply:
Shall this be postponed to a future issue? I believe it's outside the scope of this pull request.